src/language.c

```diff
@@ -238,20 +239,28 @@ static void read_lang(char *langfile)
         lskip = 1;
       }
       if (lskip) {
-        while (!strchr(lbuf, '\n') && fgets(lbuf, 511, FLANG) != NULL) {
+        while (!strchr(lbuf, '\n') && fgets(lbuf, sizeof lbuf, FLANG) != NULL) {
           lline++;
         }
         /* fgets == NULL means error or empty file, so check for error */
         if (ferror(FLANG)) {
           putlog(LOG_MISC, "*", "LANG: Error reading lang file.");
         }
-        lline++;
         lnew = 1;
         continue;
       }
-      strcpy(ltext, strchr(lbuf, ',') + 1);
-    } else
-      strcpy(strchr(ltext, 0), lbuf);
+      if ((ctmp = strchr(lbuf, ',')))
+        strcpy(ltext, strchr(lbuf, ',') + 1);
```

Comment:
why isn't the 2nd argument just ctmp?

Reply:
you mean `ctmp + 1` (stripping the leading comma). ty, simplified that one.